readthedocs/templates/projects/integration_form.html

```diff
@@ -18,6 +18,6 @@
       action="{% url 'projects_integrations_create' project_slug=project.slug %}">
     {% csrf_token %}
     {{ form.as_p }}
-    <input type="submit" value="{% trans "Add integration" %}">
+    <input type="submit" value="{% trans "Add Integration" %}">
```

Comment:
We don't use title casing on our buttons, or at least shouldn't

Reply:
Oh! Okay. I'll revert those changes. :+1: